pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientEntity.java

```diff
@@ -71,9 +59,9 @@
 	@Column(name = "last_read_date")
 	private Date lastReadDate;
 	
-	@OneToMany( fetch = FetchType.LAZY, mappedBy = "patientId"  )
+	@OneToMany( fetch = FetchType.LAZY, mappedBy = "patientRecordId"  )
```

Comment:
I'm confused about this change and about the related change in PatientOrganizationMapEntity. A PatientOrganizationMap is/was meant to keep track of which organizations a user selected as having relevant information about the person they are trying to treat. I really do think it should map to a Patient and not a PatientRecord.